packages/code-du-travail-api/src/server/routes/search/utils.js

```diff
@@ -46,6 +48,19 @@ const mergePipe = (a, b, max_result) => {
   return mergeDuplicate(res).slice(0, max_result);
 };
 
+const getDataFromUrl = (url) => {
```

Comment:
what do you think of `extractSourceSlugFromUrl(url)` ?

Reply:
It thought about it but I'm not fond of specifying the returned prop in the function's name. Seemed a bit too specific to me. With `Data` it lets the door open to new url formats without having to change de name (let's say we end up with cdntId in the url); but it's just my opnion.
What about `ExtractDataFromUrl` ?